resources/scripts/components/elements/CodemirrorEditor.tsx

```diff
@@ -79,6 +78,31 @@ require('codemirror/mode/vue/vue');
 require('codemirror/mode/xml/xml');
 require('codemirror/mode/yaml/yaml');
 
+require('codemirror/addon/hint/css-hint');
+require('codemirror/addon/hint/html-hint');
```

Comment:
I don't think we really need css and html, do we? I'd limit it to configuration and scripting languages common in available eggs.

Reply:
We supported those languages already so I added the linting for them, some people probably edit those files but there are probably very few.